pkg/cmd/server/origin/master_config.go

```diff
@@ -175,16 +177,18 @@ func BuildMasterConfig(options configapi.MasterConfig) (*MasterConfig, error) {
 	kubeletClientConfig := configapi.GetKubeletClientConfig(options)
 
 	// in-order list of plug-ins that should intercept admission decisions (origin only intercepts)
-	admissionControlPluginNames := []string{"ProjectRequestLimit", "OriginNamespaceLifecycle", "PodNodeConstraints", "BuildByStrategy", "OriginResourceQuota"}
+	admissionControlPluginNames := []string{"ProjectRequestLimit", "OriginNamespaceLifecycle", "PodNodeConstraints", "BuildByStrategy", imageadmission.PluginName, "OriginResourceQuota"}
 	if len(options.AdmissionConfig.PluginOrderOverride) > 0 {
 		admissionControlPluginNames = options.AdmissionConfig.PluginOrderOverride
 	}
 
+	registry := quota.NewRegistryForAdmission(privilegedLoopbackOpenShiftClient, options.ImagePolicyConfig.MaxImagesBulkImportedPerRepository)
```

Comment:
We still need to differentiate between registry for resource controller and for admission plugin. In k8s, they are the same. Unfortunately, it's not the case here.